lib/clearance/rack_session.rb

```diff
@@ -21,7 +21,11 @@ def call(env)
       session = Clearance::Session.new(env)
       env[:clearance] = session
       response = @app.call(env)
-      session.add_cookie_to_headers response[1]
+
+      if session.send :authentication_performed?
```

Comment:
Is there a reason to use `send` here? Isn't this method public?

Reply:
I didn't know if you wanted to expand the public api of `Session`, if you're +1 to that I'll make it public instead